examples/dreamcast/hello/hello.c

```diff
@@ -2,25 +2,38 @@
 
    hello.c
    Copyright (C) 2001 Megan Potter
+   Copyright (C) 2024 Falco Girgis
 */
 
-#include <kos.h>
+#include <stdio.h>
 
-/* These macros tell KOS how to initialize itself. All of this initialization
-   happens before main() gets called, and the shutdown happens afterwards. So
-   you need to set any flags you want here. Here are some possibilities:
+/* #include <kos.h>
+   KOS_INIT_FLAGS(INIT_DEFAULT);
+*/
+
+/* KOS_INIT_FLAGS() tells KOS how to initialize itself and which drivers to
+   pull into your executable. All of this initialization happens before main()
+   gets called, and the shutdown happens afterwards.
+
+   Here are some possible flags to pass to KOS_INIT_FLAGS():
 
    INIT_NONE        -- don't do any auto init
    INIT_IRQ         -- Enable IRQs
    INIT_NET         -- Enable networking (including sockets)
    INIT_MALLOCSTATS -- Enable a call to malloc_stats() right before shutdown
 
-   You can OR any or all of those together. If you want to start out with
-   the current KOS defaults, use INIT_DEFAULT (or leave it out entirely). */
-KOS_INIT_FLAGS(INIT_DEFAULT | INIT_MALLOCSTATS);
+   Refer to kos/init.h and arch/init_flags.h for the full list of flags.
+   You can OR any or all of these together.
+
+   If you want to start out with the current KOS defaults, use INIT_DEFAULT.
+   You may now also omit KOS_INIT_FLAGS() altogether, which will also use
+   INIT_DEFAULT automatically, as we're doing here. This can be useful if
+   you're lazy or if you're building regular C or C++ code and don't want to
+   introduce KOS-specific DC code into the file.
```

Comment:
```suggestion
   If you wish to use the default initialization settings, specify
   INIT_DEFAULT in your KOS_INIT_FLAGS(). You may also omit KOS_INIT_FLAGS()
   from your program (as we are doing here since the above code is commented
   out), which will automatically use the default initialization settings.
```

I don't think it's necessary to have the last sentence at all, and I've reworded what you have a little bit (since it isn't like the default flags being implied without a `KOS_INIT_FLAGS` is new or anything.

Spacing might need to be adjusted here, but it should be good enough.